empiric-ui/src/pages/index.tsx

```diff
@@ -41,11 +41,14 @@ const IndexPage = () => (
       <Hero />
     </SectionContainer>
     <SectionContainer className="bg-slate-50 !pt-0">
-      <LogoGrid title="Data from" logos={publishers} />
+      <LogoGrid
+        title="Proprietary data from custom integrations"
```

Comment:
think it should be "custom integrations with" just like below it is "trusted by"